gframe/config.h

```diff
@@ -56,8 +56,6 @@ inline int myswprintf(wchar_t(&buf)[N], const wchar_t* fmt, TR... args) {
 }
 
 #include <irrlicht.h>
-#include <GL/gl.h>
-#include <GL/glu.h>
```

Comment:
just change in place, do not move to other place and do not add extra empty lines